gslib/tests/test_rpo.py

```diff
@@ -102,7 +144,7 @@ def _verify_get_returns_default_or_none(self, bucket_uri):
 
   @SkipForXML('RPO only runs on GCS JSON API.')
   def test_get_returns_default_for_dual_region_bucket(self):
-    bucket_uri = self.CreateBucket(location='us')
+    bucket_uri = self.CreateBucket(location='nam4')
```

Comment:
Curious about this change.

Reply:
Test tries to create dual region bucket but it is passing the zone `us`  which creates a `multi-region` bucket, the test was passing due to only rpo config for `multi-region` bucket is `DEFAULT`, [reference](https://cloud.google.com/storage/docs/json_api/v1/buckets/patch), so when the assertion is made, returns `DEFAULT`, so I changed the code to actually create `dual-region` bucket using a [predefined dual-region](https://cloud.google.com/storage/docs/locations#predefined-dr)

Reply:
got it, thanks